### DISCUSSION TITLE
Hi Sir,This my new github account

### DISCUSSION THREAD

**Discussion by haikaru17** [ROM-builders/temporary#20744] [Q&A]
Hi sir...
This is my new account.For previous account username is hklknz,i changed because my email got hacked,you can delete my previous account & my branch named :
- tissot-bananadroid-13-hklknz
- mido-bananadroid-13-hklknz
- rova-banandroid-13-hklknz
- mido-xdroidoss-twelve-hklknz
- mido-xdroid-oss-twelve-hklknz
- hklknz

• for my trees here

• device tree :
https://github.com/bananadroid-devices/device_xiaomi_tissot/tree/13-treble

• common device tree :
https://github.com/bananadroid-devices/device_xiaomi_msm8953-common/tree/13

• vendor tree :
https://github.com/bananadroid-devices/vendor_xiaomi_tissot/tree/13

• common vendor tree :
https://github.com/bananadroid-devices/vendor_xiaomi_msm8953-common/tree/13

• kernel tree :
https://github.com/ihklknz/kernel_xiaomi_msm8953/tree/kizuna-treble

Thanks.
I hope you and all team have a nice day everytime

**Comment by Apon77** [comment]
Done
